infra/src/main/kotlin/com/cd18/infra/persistence/model/PerformanceHourlyViewStat.kt

```diff
@@ -0,0 +1,30 @@
+package com.cd18.infra.persistence.model
+
+import com.cd18.infra.persistence.config.model.BaseTimeEntity
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.LocalDate
+
+@Entity
+@Table(name = "performance_hourly_view_stat")
+class PerformanceHourlyViewStat(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Long = 0,
+    @Column(name = "statistic_date")
+    var statisticDate: LocalDate,
+    @Column(name = "statistic_hour")
+    var statisticHour: Int,
+    @Column(name = "perf_id")
+    var performanceInfoId: Long,
+    @Column(name = "view_count", nullable = false)
+    val viewCount: Long,
```

Comment:
```suggestion
    var viewCount: Long,
```
viewcount는 변경될 수 있으니 var

Reply:
배치로 통계낸 데이터가 쌓이는 구조라 변경될 가능성이 없다고 봐야할 것 같습니다!  
추후에 성능 최적화를 위해 chunk를 나눠서 데이터를 나눠서 통계 정보를 생성하는 경우에는 변경이 필요할 수 있을 것 같습니다🙃